README.md

```diff
@@ -36,3 +36,23 @@ jaeger:
       - 'healthcheck'
       - 'metrics'
 ```
+
+
+## Name generation options
+
+You can specify just the suffix, if you name generator service is named as `jaeger.name.generator.*` or if you have any other naming scheme you can put the whole name into the configuration.
```

Comment:
```suggestion
You can configure custom name generators based on regular expression pattern, which will be evaluated for operation name.

Configuration for this feature looks like key-value list, where key - regexp pattern, value - custom name generator DI service id (see details below).

Name generator should implement an 'TODO' interface.
As a custom name generator you can specify a full DI service id, or just the suffix if your name generator service is named as `jaeger.name.generator.*`.
```

My suggestion can be rephrased because it may contain some issues with grammar and github have limited support for suggestion based on one line from file.